ProcessLib/ProcessVariable.cpp

```diff
@@ -56,18 +56,23 @@ ProcessVariable::ProcessVariable(
                     //! \ogs_file_param{prj__process_variables__process_variable__boundary_conditions__boundary_condition__geometry}
                     bc_config.getConfigParameter<std::string>("geometry");
 
-            GeoLib::GeoObject const* const geometry =
-                geometries.getGeoObject(geometrical_set_name, geometry_name);
-
-            if (! geometry)
-                OGS_FATAL(
-                    "No geometry with name `%s' has been found in the "
-                    "geometrical set `%s'.",
-                    geometry_name.c_str(), geometrical_set_name.c_str());
-
-            DBUG(
-                "Found geometry type \"%s\"",
-                GeoLib::convertGeoTypeToString(geometry->getGeoType()).c_str());
+            auto const full_geometry_name =
+                geometrical_set_name + "_" + geometry_name;
```

Comment:
IMHO it should be documented that `_` is used as a separator here. Maybe it should be also enforced that neither the set name nor the geometry name contain underscores.

Reply:
--> discussion.
I'd like to switch to proper mesh names (for the bulk mesh and boundary meshes) rather soon, s.t. we can use the vtu file names or mesh names from the MED files.